package.json

```diff
@@ -26,7 +26,7 @@
     "update-snapshots": "yarn cmsds test ./packages --updateSnapshot",
     "test-coverage": "yarn cmsds test ./packages --collectCoverage",
     "posttest": "yarn lint",
-    "lint": "yarn cmsds lint ./ --ignorePatterns '**/node_modules/**' '**/dist/**' '**/helpers/**' '**/__tests__/**' 'tmp/**' '**/types/**' 'examples/create-react-app-typescript/**' 'examples/create-react-app/**' 'storybook-static/**' 'packages/coverage/**'",
+    "lint": "yarn cmsds lint ./ --ignorePatterns '**/node_modules/**' '**/dist/**' '**/helpers/**' '**/__tests__/**' 'tmp/**' '**/types/**' 'examples/create-react-app-typescript/**' 'examples/create-react-app/**' 'storybook-static/**' 'packages/coverage/**' 'packages/design-system-tokens/lib/types.ts'",
```

Comment:
unfortunately this is the only way around this until we move off gulp (gulp-prettier) or they upgrade their version of prettier to 2.2 to support template literal types.. I think I'll submit a PR to them to fix this actually.